src/Intouch/LaravelNewrelic/LaravelNewrelicServiceProvider.php

```diff
@@ -98,7 +100,7 @@ function ( $request, $response ) use ( $me, $app )
      */
     public function getTransactionName( $request, $response, $app )
     {
-        $nameProvider = $app['config']['laravel-newrelic::name_provider'];
+        $nameProvider = $app['config']->get('laravel-newrelic::name_provider');
```

Comment:
@lukewaite I haven't tagged a new v yet, just merged as I work on the update for L5.  Will fix if it's broken before tagging, but at least people can composer install in an L5 project for now (@dev-master)